v2_1/ws/rest/src/sdmx-rest.yaml

```diff
@@ -222,6 +222,21 @@ paths:
           $ref: '#/components/responses/503'
           
 components:
+  schemas:
+    SdmxPeriod:
+      type: string
+      pattern: '^\d{4}-?((\d{2}(-\d{2})?)|A1|S[1|2]|Q[1-4]|T[1-3]|M(0[1-9]|1[0-2])|W(0[1-9]|[1-4][0-9]|5[0-3])|D(0[0-9][1-9]|[1-2][0-9][0-9]|3[0-5][0-9]|36[0-6]))?$'
```

Comment:
The pattern is a bit too relaxed, compared to what is allowed by ISO8601. For example, it allows wrong periods like the following:
202012
2020-M12
2020-99
2020-99-99

I understand that covering all ISO8601 with a single regex might be a bit challenging; hence, If we are not seeking an exhaustive validation, then we may proceed with this.

Reply:
OAS3 allows `anyOf` so we can use different patterns. 

OAS3 `date` and `date-time` implies RFC 3339 https://swagger.io/docs/specification/data-models/data-types/

IMHO we can skip all the quirks and limit ISO 8106 to the RFC 3339 profiles.